diem-unified-charter.md

```diff
@@ -27,29 +26,30 @@ This can be a separate entity from the emblem issuer.
 
 "To validate an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
 Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific verifier upon request.
-However, cryptographic verification can be optional as long is the emblem can be correctly interpreted.
+Cryptographic verification can be optional as long is the emblem can be correctly interpreted.
 Therefore, to be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
 
-Assets bearing to digital emblems can be analog (e.g., a vehicle) or digital (e.g., a web page), however, there need to be a digital representation that is used for validation using digital communication.
+Digital emblems can be bound to entities that are analog (e.g., a vehicle) or digital (e.g., a web page), however, there needs to be a digital representation that is used for validation using digital communication.
 
 Digital emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will affect how a validator interprets and trusts the assertions made within the emblem.
-The communication between the asset and the validator will influence requirements on how validators retrieve digital emblems for the asset. 
+The communication between an entity that bears an emblem and a validator will influence requirements on how validators retrieve digital emblems. 
 These requirements will be key drivers in the development of a digital emblem architecture.
 
 The architecture, specified by this group, must be flexible enough to support multiple serialization formats, securing mechanisms, discovery mechanisms, and use cases. 
 
 # Initial Scope
 
-The working group will focus its initial work on assets that are discovered by use of digital communication such as use of DNS or at a well-known location on a host identified by a hostname or IP address. 
-These discovery mechansisms have the unique challenge of the validator remaining unknown to the bearer or holder.
-Specifically the design of other discovery mechanisms using proximity-based protocols such as  QRCodes, NFC, or Bluetooth is out-of-scope.
+The working group will focus its initial work on emblems that are discovered by use of digital communication such as use of DNS or at a well-known location on a host identified by a hostname or IP address.
+Additionally, these discovery mechansisms need to address the unique challenge of the validator remaining unknown to the operator of the entity that bears that emblem.
```

Comment:
We still have the word operator here, how is the operator related to the holder or presenter of the emblem?
https://github.com/OR13/diem-draft-charter/issues/15